ftdetect/set_repl_cmd.vim

```diff
@@ -40,6 +43,11 @@ if has('nvim')
           \ if executable('R') |
           \   call neoterm#repl#set('R') |
           \ end
+    " PARI/GP
```

Comment:
Yes, it's a computer algebra system focused on number theory: http://pari.math.u-bordeaux.fr/
However, I did not mean it to be part of this pull request. I will remove it and make a separate pull request later.